projects/plugins/jetpack/_inc/jetpack-deactivate-dialog.js

```diff
@@ -28,10 +29,10 @@
 						// NodeList is static, we need to modify this in the DOM
 
 						$( '#TB_window' ).addClass( 'jetpack-disconnect-modal' );
-						deactivationModalCentralize();
+						window.deactivationModalCentralize();
```

Comment:
These functions all get created as `window.xyz` in this file, changed the invocations to match to avoid a bunch of `no-undef` lints.